src/main/java/com/pinterest/secor/message/Message.java

```diff
@@ -34,21 +34,23 @@
     private long mOffset;
     private byte[] mKafkaKey;
     private byte[] mPayload;
+    private Long mTimestamp;
 
     protected String fieldsToString() {
         return "topic='" + mTopic + '\'' +
                ", kafkaPartition=" + mKafkaPartition +
                ", offset=" + mOffset +
                ", kafkaKey=" + new String(mKafkaKey) +
-               ", payload=" + new String(mPayload);
+               ", payload=" + new String(mPayload) +
+               ", mTimestamp=" + mTimestamp;
```

Comment:
I don't think you need to use 'mTimestamp' in the printout, the other internal variables (payload, kafkaKey) didn't do that either.